L1Trigger/CSCTriggerPrimitives/python/cscTriggerPrimitiveDigis_cfi.py

```diff
@@ -11,7 +11,7 @@ def _modifyCscTriggerPrimitiveDigisForRun2( object ) :
     """
     object.debugParameters = True
     object.checkBadChambers = False
-    object.commonParam.isSLHC = True
+    object.commonParam.isSLHC = False
```

Comment:
Good point. Since the "eras" are now available for upgrade scenarios, I can add appropriate  customization functions at the bottom of this file.